src/playerClass.ts

```diff
@@ -0,0 +1,22 @@
+import { Pokemon } from "./pokemonClass";
+
+export class Player {
+  private _name: string;
+  private _pokemons: Pokemon[];
```

Comment:
Maybe we can add an attribute to store a currently active pokemon, later we will use it in the fight. 
`private _activePokemon: number;`
And during the initialization
`this._activePokemon = 0;`

Reply:
And of course get and set methods for this parameter

Reply:
It's a good idea. I'll add this to the class.